packages/backup-restore-operator/charts/templates/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+kind: ClusterRoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: backup-restore-operator-installer
```

Comment:
I think all mentions of `backup-restore-operator` should be changed to `{{ .Chart.Name }}`